extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -17,6 +17,8 @@ import * as which from 'which';
 import { promises as fs } from 'fs';
 import { ISqlProject, SqlTargetPlatform } from 'sqldbproj';
 
+const SqlDWEngineEdition = 6;
```

Comment:
can you also add a comment here about where this number can be found in the ADS and vscode codebases?

Reply:
What I'd suggest instead - 

1. Add https://github.com/Microsoft/azuredatastudio/blob/main/src/sql/azdata.d.ts#L539 to https://github.com/Microsoft/vscode-mssql/blob/8ff6c2fdc3e8f5cecd3aed3ec2bad9d9c10073c0/typings/vscode-mssql.d.ts but make it a const enum
2. Update the vscode-mssql typings file in the ads repo and then reference that

That enum should be there anyways for other use, and that gets around the trickiness of the ts compiler not letting you reference stuff from an "import type" (even if it's a const enum)

Reply:
Created https://github.com/microsoft/vscode-mssql/issues/17448 to work on the first part.

Reply:
Thanks, I'll sign off now so you aren't blocked once you get this in since that should be straightforward. Just wanted to make sure we did this now to save having to go back and fix it 😄

Reply:
Yep completely agreed, I did think about this while working on vscode side of changes. Better late than super late :-D .